packages/docs-site/.vitepress/config.ts

```diff
@@ -0,0 +1,129 @@
+import { defineConfig } from "vitepress";
+
+export default defineConfig({
+  ignoreDeadLinks: true,
+  outDir: "build",
+  title: "Penrose",
+  description:
+    "Create beautiful diagrams just by typing math notation in plain text.",
+  head: [["link", { rel: "icon", href: "/img/logo.svg" }]],
+  cleanUrls: "with-subfolders",
+  themeConfig: {
+    logo: "img/favicon.ico",
+    nav: [
+      {
+        text: "Learn Penrose",
+        link: "/docs/tutorial/welcome",
+        activeMatch: "/docs/tutorial",
+      },
+      { text: "Documentation", link: "/docs/ref", activeMatch: "/docs/ref" },
+      { text: "Try Penrose", link: "/try/index.html" },
```

Comment:
looks like this link is dead. I tried dynamically generating the link, or creating a blank component that routes to the static HTML page. Neither worked out. Given that we can only deploy one page per repo on cloudflare, changing to absolute links under `penrose.cs.cmu.edu` will break our preview deployments. Feeling a little stuck here.